src/host/ConsoleArguments.hpp

```diff
@@ -32,7 +32,6 @@ class ConsoleArguments
     [[nodiscard]]
     HRESULT ParseCommandline();
 
-    bool IsUsingVtPipe() const;

```

Comment:
Declared but never implemented.

Reply:
Haha, that was vestigial from a much earlier implementation of conpty. good eye on finding that!